docs/security/https/README.md

```diff
@@ -14,20 +14,20 @@ At this point, it has to be a manual thing, and I've been running it in boot2doc
 My process is as following:
 
     $ boot2docker ssh
-    $$ git clone https://github.com/docker/docker
-    $$ cd docker/docs/articles/https
-    $$ make cert
+    root@f38c87f2a42d:/# git clone https://github.com/docker/docker
+    root@f38c87f2a42d:/# cd docker/docs/articles/https
+    root@f38c87f2a42d:/# make cert
```

Comment:
oh! In this document it probably needs to be `root@boot2docker:/#` (as it mentions that the hostname is `boot2docker` below)